node-operators/appchain-node/rpc-systemd.md

```diff
@@ -0,0 +1,265 @@
+    ```
+
+!!! note
+    Optimized binary versions for [Skylake](https://www.intel.com/content/www/us/en/products/platforms/details/skylake-u-y.html){target=\_blank} and [Zen3](https://www.amd.com/en/technologies/zen-core){target=\_blank} architectures are also available in the [releases](https://github.com/moondance-labs/tanssi/releases/latest){target=\_blank} page.
+
+## Download the Relay Chain Specs File {: #download-relay-specs }
+
+The node binary file includes also the necessary code to run a relay chain node. When launching your Appchain's node, it will also be required to provide the relay chain's specification file as a parameter. 
+
+Download the relay chain specification file executing:
+
+```bash
+wget https://raw.githubusercontent.com/papermoonio/external-files/main/Moonbeam/Moonbase-Alpha/westend-alphanet-raw-specs.json
+```
+
+## Setup the Systemd Service {: #setup-systemd-service }
+
+[Systemd](https://systemd.io/){target=\_blank} is a management system for Linux systems that manages services (daemons in Unix-like systems jargon), starting them automatically when the computer starts or reboots, or restarting them upon unexpected failures.
+
+It is a good practice to have the service running with its own non-root account and grant that account writing access to a specific directory. Run the following commands to configure a new account and the directory:
+
+Create a new account to run the service:
+
+```bash
+adduser appchain_node_service --system --no-create-home
+```
+
+Create a directory to store the required files and data:
+
+```bash
+mkdir /var/lib/appchain-data
+```
+
+Set the folder's ownership to the account that will run the service to ensure writing permission:
+
+```bash
+sudo chown -R appchain_node_service /var/lib/appchain-data
+```
+
+And finally, move the binary and the relay chain spec to the folder:
+
+```bash
+mv ./container-chain-template-*-node /var/lib/appchain-data && \
+mv ./westend-alphanet-raw-specs.json /var/lib/appchain-data
+```
+
+!!! note
+    To keep all the necessary files grouped in the same directory, it is recommended to also copy there your Appchain's specification file.
+
+### Create the Systemd Service configuration file {: #create-systemd-configuration }
+
+The next step is to create the Systemd configuration file. 
+
+You can create the file by running the following command:
+
+```bash
+sudo touch /etc/systemd/system/appchain.service
+```
+
+Now you can open the file using your favorite text editor (vim, emacs, nano, etc) and add the configuration for the service:
+
+```bash
+[Unit]
+Description="Appchain systemd service"
+After=network.target
+StartLimitIntervalSec=0
+
+[Service]
+Type=simple
+Restart=on-failure
+RestartSec=10
+User=appchain_node_service
+SyslogIdentifier=appchain
+SyslogFacility=local7
+KillSignal=SIGHUP
+ExecStart=/var/lib/appchain-data/container-chain-template-APPCHAIN_TYPE-node \
+--chain=YOUR_APPCHAIN_SPECS_FILE_LOCATION \
```

Comment:
Use the same structure as with [Docker](https://docs.tanssi.network/node-operators/appchain-node/rpc-docker/#start-up-command).

That is:
- Explain Data they have to modify and then show the content of the file. Also, put the content of the flags in the same order as Docker